src/lbfgs.jl

```diff
@@ -18,8 +18,10 @@ The keyword arguments may include
 - `rtol::T = √eps(T)`: relative tolerance, the algorithm stops when ||∇f(xᵏ)|| ≤ atol + rtol * ||∇f(x⁰)||.
 - `max_eval::Int = -1`: maximum number of objective function evaluations.
 - `max_time::Float64 = 30.0`: maximum time limit in seconds.
+- `τ₁::T = T(0.9999)`: slope factor in the Wolfe condition when performing the line search.
+- `bk_max:: Int = 25`: maximum number of backtracks when performing the line search.
 - `verbose::Int = 0`: if > 0, display iteration details every `verbose` iteration.
-- `mem::Int = 5`: memory parameter of the `lbfgs` algorithm.
```

Comment:
You can keep `mem` in the docstring as it is used by the function `lbfgs`.

Oh, I actually realize that it's not and this is a bug
https://github.com/ProofOfConceptForJuliSmoothOptimizers/JSOSolvers.jl/blob/691905208b88a7e1c30ef8027ccf5f4922ab3190/src/lbfgs.jl#L84
should have `mem` in the `kwargs` of the function `lbfgs` and pass it to the `LBFGSSolver`

Reply:
Oh I see! No problem, I will fix it.